### DISCUSSION TITLE
Please upgrade your Copilot extension to continue using this service.

### DISCUSSION THREAD

**Discussion by 1986791206** [community/community#18393] [Copilot Conversations] [answered]
[INFO] [default] [2022-06-12T02:05:22.610Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-12T02:05:22.845Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 235 ms
[INFO] [fetch] [2022-06-12T02:05:22.847Z] Please upgrade your Copilot extension to continue using this service.

**Comment by davecheney** [comment, marked as answer]
You are using version 1.17.5850, you need to upgrade. Please see https://github.com/github-community/community/discussions/16815
